showmehow/lessons.json

```diff
@@ -184,38 +184,38 @@
                         "reply": "Interesting.... I wonder what we can play around with here...",
-                        "move_to": "disable_social_bar"
+                        "move_to": "get_clock_format"
                     },
                     "failure": {
-                        "reply": "I want to know about the settings available for org.gnome.shell in sorted order? Can you list-keys on org.gnome.shell using gsettings and then pipe (|) the output to `sort`? Try:\n$ gsettings list-keys org.gnome.shell | sort"
+                        "reply": "I want to know about the settings available for org.gnome.desktop.interface in sorted order? Can you list-keys on org.gnome.desktop.interface using gsettings and then pipe (|) the output to `sort`? Try:\n$ gsettings list-keys org.gnome.desktop.interface | sort"
                     }
                 }
             },
-            "disable_social_bar": {
-                "task": "Lets disable the social bar! The social bar's setting is 'enable-social-bar' and it is a `boolean` setting, meaning it is either 'true' or 'false'. I checked `gsettings help set` for you and it said you can use the `set` command to change a setting with gsettings [--schemadir SCHEMADIR] set SCHEMA[:PATH] KEY VALUE. Your job is to provide KEY and VALUE, those being 'enable-social-bar' and 'false' to `gsettings set`.",
+            "get_clock_format": {
+                "task": "Lets learn how to change the clock format using 'gsettings'. The two avaliable formats are '12h' or '24h'. Lets start by finding out what format is set in the 'clock-format' key, by using the `get` command. The format should be the following. `gsettings get <our-schema> <clock-schema-key>`.",
                 "effects": {
                     "success": {
-                        "reply": "There we go! No Facebook icon!",
-                        "move_to": "enable_social_bar"
+                        "reply": "That's the current clock format on this machine.",
+                        "move_to": "change_clock_format"
                     },
                     "failure": {
-                        "reply": "Hmm, I thought you'd change the value of org.gnome.shell e'enable-social-bar' to false, but it is still true. Remember to change a gsettings value you can use:\n$ gsettings set KEY VALUE"
+                        "reply": "Thats not what we want. Try replacing '<our schema>' with 'org.gnome.desktop.interface' and 'clock-schema-key' with 'clock-format' in the following command `gsettings get <our-schema> <clock-schema-key>`."
                     }
                 }
             },
-            "enable_social_bar": {
-                "task": "Now lets change it back to true",
+            "change_clock_format": {
+                "task": "Now lets change the 'clock-format' to the '24h' format. This is done in the same way as we just did to view the current clock format, but instead we will be using the 'set' command.",
```

Comment:
As discussed let's make this independent of the current format set and rephrase to something like: "...there you see your current format - now try flipping it, to the other format...".